bdb/bdb_int.h

```diff
@@ -767,7 +767,7 @@ typedef struct table_version_cache
 {
     int sz;
     unsigned long long *entries;
-} table_version_cache;
+} table_version_cache_t;
```

Comment:
Sorry, this was my bad--maybe better to just remove this typedef altogether because it isn't used?